catboost/private/libs/target/data_providers.cpp

```diff
@@ -494,8 +494,11 @@ namespace NCB {
                 << " loss/metrics require target data"
             );
         }
+        
+        bool needCheckTarget = !mainLossFunction || !IsRegressionObjective(mainLossFunction->GetLossFunction());
+        //Acept nan in MultiRMSEWithMissingValues
+        bool allowNanTarget = (mainLossFunction->GetLossFunction() == ELossFunction::MultiRMSEWithMissingValues);
```

Comment:
Sometime model does not contain loss function (for example, after `sum_models`), now it fails. Let's allow NaNs in such case.
`bool allowNanTarget = !mainLossFunction || (mainLossFunction->GetLossFunction() == ELossFunction::MultiRMSEWithMissingValues);
`

Reply:
`const bool`

Reply:
I see. 
Done.